packages/asc-ui/src/Radio/Radio.tsx

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react'
-import { Radio as RadioStyled } from '@amsterdam/asc-core'
-
+import { Radio as RadioStyled, RadioLabel, RadioItem } from '@amsterdam/asc-core'
+console.log(RadioItem);
 const Radio = (props: any) => (
-  <RadioStyled {...props} />
+  <div>
```

Comment:
Don't think you need to use div here, as RadioItem already has one.

Reply:
removed!